test/known_issues/test-inspector-cluster-port-clash.js

```diff
@@ -0,0 +1,60 @@
+// Flags: --inspect=0
+'use strict';
+const common = require('../common');
+
+// There is an issue when forking workers that try to bind a port for inspector.
+// If the port is not free the worker will fail.
+// This test should fail with:
```

Comment:
Can this comment be clarified a bit? This makes it sound like "If Node.js were working correctly and also addressing this issue, then this test should fail." Maybe instead of "should", something like "This test currently fails with:" or maybe something even more verbose like:

> With the current behavior of Node.js (at least as late as 8.1.0), this test fails with the following error:
>   AssertionError [ERR_ASSERTION]: worker 2 failed to bind port
> Ideally, there would be a way for the user to opt out of sequential port assignment and this test would then pass.